clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/heartbeat/DefaultPushJobHeartbeatSenderFactory.java

```diff
@@ -143,10 +144,11 @@ protected VeniceWriter<byte[], byte[], byte[]> getVeniceWriter(
         versionCreationResponse.getPartitionerClass(),
         versionCreationResponse.getAmplificationFactor(),
         new VeniceProperties(partitionerProperties));
-    return new VeniceWriterFactory(veniceWriterProperties).createBasicVeniceWriter(
-        versionCreationResponse.getKafkaTopic(),
-        new SystemTime(),
-        venicePartitioner,
-        versionCreationResponse.getPartitions());
+
+    return new VeniceWriterFactory(veniceWriterProperties).createVeniceWriter(
+        new VeniceWriterOptions.Builder(versionCreationResponse.getKafkaTopic()).setTime(new SystemTime())
```

Comment:
nit: not critical and thanks for changing to builder pattern. Wondering if setTime(new SystemTime()) can be the default value if not specified.

Reply:
The current default is `SystemTime.INSTANCE`  which is essentially `new SystemTime()`. So even if we do not set it should be fine. 

removing it from here. Thanks.